azure-resources/Sql/servers/recommendations.yaml

```diff
@@ -17,17 +17,17 @@
     - name: Active Geo Replication
       url: "https://learn.microsoft.com/en-us/azure/azure-sql/database/active-geo-replication-overview"
 
-- description: Auto Failover Groups for apps should include all related databases for cohesion
+- description: Use Auto Failover Groups that can include one or multiple databases, typically used by the same application
```

Comment:
Please re-word to bring the description under the 100 character limit, including spaces.

Reply:
This line is 88 characters including spaces. Not sure why this is above 100 characters?

Reply:
Word and a couple online character counters show this as 107 characters, including spaces.